examples/notebooks/voltage-regulator/publish_waveforms.ipynb

```diff
@@ -12,48 +12,118 @@
   },
   {
    "cell_type": "markdown",
-   "id": "db8657b1",
+   "id": "a0ec35e9",
    "metadata": {},
    "source": [
-    "## Set up the test result for publishing data\n",
+    "# Set up the test station and other metadata (one time setup)\n",
     "\n",
-    "First, we're going to set up our test result metadata and get everything ready to publish"
+    "Setting up the metadata for operators, test stations, hardware, etc will typically be done once. We establish 'aliases' which are human-readable strings that can be used to refer to metadata later."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "5aaa09bb",
+   "id": "72711880",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from ni.datastore import Client, Operator, TestResult, TestStation\n",
+    "from ni.datastore import (\n",
+    "    Client,\n",
+    "    HardwareItem,\n",
+    "    Operator,\n",
+    "    SoftwareItem,\n",
+    "    TestStation,\n",
+    "    Uut,\n",
+    "    UutInstance,\n",
+    ")\n",
+    "\n",
+    "# Constants\n",
+    "ALIAS_JAMES_BOWERY = \"Operator_BoweryJ\"\n",
+    "ALIAS_TEST_STATION_12 = \"TestStation_12\"\n",
+    "ALIAS_UUT_A861_42367 = \"UUT_A861_42367\"\n",
+    "ALIAS_SOFTWARE_ITEM_WINDOWS_10 = \"Windows_10.0.19044\"\n",
+    "ALIAS_SOFTWARE_ITEM_PYTHON_3_12 = \"Python_3.12\"\n",
+    "ALIAS_HARDWARE_ITEM_PXIe_5171 = \"HardwareItem_PXIe-5171\"\n",
     "\n",
     "client = Client()\n",
     "\n",
+    "# Create alias for Operator \"James Bowery\"\n",
     "operator = Operator(operator_name=\"James Bowery\", role=\"Test Architect\")\n",
-    "operator_id = client.create_operator(operator)\n",
+    "client.create_operator(operator)\n",
+    "alias_james_bowery = client.create_alias(ALIAS_JAMES_BOWERY, operator)\n",
     "\n",
+    "# Create alias for Test Station \"TestStation_12\"\n",
     "test_station = TestStation(test_station_name=\"TestStation_12\")\n",
-    "test_station_id = client.create_test_station(test_station)\n",
+    "client.create_test_station(test_station)\n",
+    "alias_test_station_12 = client.create_alias(ALIAS_TEST_STATION_12, test_station)\n",
+    "\n",
+    "# Create alias for UUT instance \"A861-42367\"\n",
+    "uut = Uut(model_name=\"NI-6508\", family=\"Digital\")\n",
+    "uut_id = client.create_uut(uut)\n",
+    "uut_instance = UutInstance(uut_id=uut_id, serial_number=\"A861-42367\")\n",
```

Comment:
uut instance is not part of the pre-run setup. Like the test result, and step, this is dynamic with each run. Consider maving uut_instance creation down to publish step. We probably don't need an alias for uut instance either.